connectors/src/lib/temporal_monitoring.ts

```diff
@@ -78,6 +77,14 @@ export class ActivityInboundLogInterceptor
       );
     }, this.context.info.startToCloseTimeoutMs);
 
+    // We already trigger a monitor after 20 attempts, but when the pod carshed (eg: OOM or segfault), the attempt never get logged.
+    // So we add a monitor the attempt count before the activity starts.
+    if (this.context.info.attempt > 25) {
+      this.logger.error(
```

Comment:
This means we're going to get one such monitor for _every_ try no ?

Reply:
so lets add a `=== 25` ? With this, we will know that THIS activity is extremely likeley to be the one crashing the pod.

Reply:
If I understand correctly, this is mostly about logging at the beginning of the activity vs logging on failure. Why not move / change the existing log instead of adding a panic one ?

Reply:
@fontanierh actually hesitated, because the other logs actually gives you the error, and this one does not.
So while they are close, they serve different purpose.
The existing one helps debug an activity that failed more than 20 times, and this new one tells you "hey, I am an activity that is probably very problematic, but we don't know why".

We could add a %10 on the attempt number and panic only once every 10 failures after 20 attempts?